src/lib/_global-variables.scss

```diff
@@ -0,0 +1,20 @@
+@import "@patternfly/patternfly/base/patternfly-variables.scss";
```

Comment:
:eyes: what happened here? This shouldn't be in git, but checked out from cockpit in `Makefile`